Source/Core/Core/HW/EXI_DeviceSlippi.cpp

```diff
@@ -188,7 +188,7 @@ std::unordered_map<u8, std::string> CEXISlippi::getNetplayNames()
 		for (auto it = netplayPlayers.begin(); it != netplayPlayers.end(); ++it)
 		{
 			auto player = *it;
-			u8 portIndex = netplay_client->FindPlayerPad(player);
+			s8 portIndex = netplay_client->FindPlayerPad(player);
 			if (portIndex < 0)
```

Comment:
a u8 can never be less than 0... this changes the way this works, but clearly does what the code intended (risky?)